go/vt/servenv/http.go

```diff
@@ -0,0 +1,39 @@
+package servenv
+
```

Comment:
There is a header missing here, no linter yet 👀

Reply:
Done!